HMD Calibration crashes pupil service

Stopping the hmd calibration causes the pupil service to crash.

relevant Log:
2021-03-17 13:55:14,149 - service - [ERROR] launchables.service: Process Service crashed with trace:
Traceback (most recent call last):
  File "launchables\service.py", line 287, in service
  File "calibration_choreography\hmd_plugin.py", line 86, in on_notify
  File "calibration_choreography\base_plugin.py", line 512, in on_notify
  File "calibration_choreography\base_plugin.py", line 569, in _perform_stop
AttributeError: 'types.SimpleNamespace' object has no attribute 'trigger_main_window_redraw'

operating system: windows 10
version: v3.2-16

steps to reproduce:
- start HMD3DChoreographyPlugin
- start calibration "calibration.should_start"
- add some calibration points
- stop calibration "calibration.should_stop"

--- a/pupil_src/shared_modules/calibration_choreography/base_plugin.py
+++ b/pupil_src/shared_modules/calibration_choreography/base_plugin.py
@@ -565,8 +565,10 @@ class CalibrationChoreographyPlugin(Plugin):
             )
             return
 
-        # Reset the main window size to trigger a redraw with correct size and scale
-        self.g_pool.trigger_main_window_redraw()
+        if self.g_pool.app == "capture":
+            # Reset the main window size to trigger a redraw with correct size and scale
+            # Only run in Capture to fix https://github.com/pupil-labs/pupil/issues/2119
+            self.g_pool.trigger_main_window_redraw()
 
         current_mode = self.__current_mode
         pupil_list = self.__pupil_list

